src/test/java/loci/common/utests/URLHandleTest.java

```diff
@@ -66,6 +69,10 @@ public void setup() throws IOException {
     tmpFile.deleteOnExit();
     FileOutputStream out = new FileOutputStream(tmpFile);
     out.write("hello, world!\n".getBytes(Constants.ENCODING));
+    byte[] emptyBuffer = new byte[EMPTY_BUFFER_SIZE];
+    Arrays.fill(emptyBuffer, (byte) 0);
```

Comment:
This is a no-op: new bytes arrays are already zeroed.

Reply:
Fixed in f78eed7